tests/tests/test-subscription.ts

```diff
@@ -177,54 +179,37 @@ describeWithMoonbeam(
         "0x0000000000000000000000000000000000000000000000000000000000000000",
         "0x0000000000000000",
       ]);
-      setTimeout(done, 10000);
-    }).timeout(20000);
```

Comment:
is this why tests are taking less time now?

Reply:
Yes. A bit of context to understand why those timeouts were there in the first place: the tests passed **locally** without them, but timed-out on github actions. Now that we are using different CI machine, I believe the timeouts are not needed anymore.

They were artifically adding 10 seconds per step to make sure they passed on github, so obviously if we remove them that's 10 seconds saved per step.

Reply:
It was not really related to the github server, but maybe to the node version being used. However It is not obvious why they would have failed without the added delay.